staging/src/k8s.io/apiserver/pkg/server/options/serving_unix.go

```diff
@@ -29,3 +29,9 @@ func permitPortReuse(network, addr string, conn syscall.RawConn) error {
 		syscall.SetsockoptInt(int(fd), syscall.SOL_SOCKET, unix.SO_REUSEPORT, 1)
 	})
 }
+
+func permitAddressReuse(network, addr string, conn syscall.RawConn) error {
+	return conn.Control(func(fd uintptr) {
+		syscall.SetsockoptInt(int(fd), syscall.SOL_SOCKET, unix.SO_REUSEADDR, 1)
```

Comment:
we don't want to track potential errors?

https://golang.org/pkg/syscall/#SetsockoptInt
```
func SetsockoptInt(fd, level, opt int, value int) (err error)
```

Reply:
yeah maybe

Reply:
done